src/main/java/net/openhft/chronicle/wire/Base40IntConverter.java

```diff
@@ -25,10 +25,11 @@
 /**
  * Unsigned 32-bit number.
  */
+@Deprecated(/* to remove in x.25 */)
 public class Base40IntConverter implements IntConverter {
     public static final int MAX_LENGTH = IntConverter.maxParseLength(40);
     public static final Base40IntConverter INSTANCE = new Base40IntConverter();
-    private static final String CHARS = ".ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789:-+";
```

Comment:
this is a breaking change? i.e. encoded ints will not be compatible before/after? If so, we should either be able to read the old style identifiers or at least warn the user if we encounter an id with a ":"?

Reply:
What isn't clear in this is that using colon meant the value couldn't be read safely as it was (in TextWire or YamlWire). This only worked if you had a value that didn't contain that character.